src/mypy.ini

```diff
@@ -27,13 +27,18 @@ disallow_untyped_defs = True
 [mypy-cephadm.*]
 disallow_untyped_defs = True
 
+[mypy-crash.*]
+disallow_untyped_defs = True
+
 [mypy-devicehealth.*]
 disallow_untyped_defs = True
 
+[mypy-hello.*]
+disallow_untyped_defs = True
+
 [mypy-iostat.*]
 disallow_untyped_defs = True
 
->>>>>>> 514d3b6785... mgr/rook: disallow_untyped_defs = True
```

Comment:
@sebastian-philipp i am sorry for introducing this change. this change is included by #39198.

Reply:
no worries. On the contrary, thanks for merging for lots of cephadm PRs yesterday